rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1,4 +1,4 @@
-#include "game_list_frame.h"
+﻿#include "game_list_frame.h"
```

Comment:
why this change?

Reply:
I didn't change it. No idea why it is reported